sklearn/ensemble/stacking.py

```diff
@@ -0,0 +1,166 @@
+"""Stacking API"""
+
+# Author: Caio Oliveira <caioaao@gmail.com>
+# License: BSD 3 clause
+
+from ..base import (BaseEstimator, TransformerMixin, MetaEstimatorMixin)
+from ..model_selection import cross_val_predict
+from ..pipeline import (make_union, make_pipeline)
+from ..preprocessing import FunctionTransformer
+
+
+class BlendedEstimator(BaseEstimator, MetaEstimatorMixin, TransformerMixin):
+    """Transformer to turn estimators into blended estimators
+
+    This is used for stacking models. Blending an estimator prevents data leaks
```

Comment:
I think you should define what blending is. It's a simple concept, but shouldn't be presumed knowledge.